cmd/jira/create.go

```diff
@@ -0,0 +1,100 @@
+package jira
+
+import (
+	"fmt"
+	"github.com/andygrunwald/go-jira"
+	"github.com/openshift/osdctl/pkg/utils"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"strings"
+)
+
+const (
+	TeamNameKey        = "jira_team"
+	TeamLabelKey       = "jira_team_label"
+	BoardIdLabel       = "jira_board_id"
+	DefaultDescription = ""
+	DefaultTicketType  = "Task"
+	DefaultProject     = "OSD"
+	SprintState        = "active"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create <title>",
```

Comment:
Maybe we could rename this to more align with the intent we had in our team discussion to easily track work that it otherwise untracked. I like the name "quick-task" since the issue type this creates is a Task.

Reply:
I think it makes sense to make a new `ticket` subcommand family that handles any create, delete, modify actions against individual jira issues. Users could then invoke them with 

```
$ osdctl jira ticket create
 ...
 $ osdctl jira ticket delete
 ...
 $ osdctl jira ticket modify
 ...
 # (etc)
 ```

Reply:
I like that for a next iteration that builds off this work. For now though, a generic `jira ticket create` will need to be less prescriptive than the intent behind the team's request.